doc/api/modules.md

```diff
@@ -491,8 +491,8 @@ added: v0.0.1
 
 * {string}
 
-The file name of the current module. This is the resolved absolute path of the
-current module file.
+The file name of the current module. This is current module file's absolute path
```

Comment:
Should "This is current" be "This is the current"?

Reply:
Yes, thanks, I'll fix that.